src/osemgrep/cli_lsp/Lsp_CLI.ml

```diff
@@ -39,4 +39,5 @@ let cmdline_info : Cmd.info = Cmd.info "semgrep lsp" ~doc ~man
 
 let parse_argv (argv : string array) : conf =
   let cmd : conf Cmd.t = Cmd.v cmdline_info cmdline_term in
+  Fmt_tty.setup_std_outputs ?style_renderer:None ();
```

Comment:
You can use CLI_common.setup_logging with the conf.common argument.
See what we do in Scan_subcommand.ml or Ci_subcommand.ml.
But actually this should be done in Lsp_subcommand.ml, not Lsp_CLI.ml which should just parse the cmdline arguments and nothing more (that is return a 'conf' from an argv).